Source/SwiftyDropbox/Platform/SwiftyDropbox_iOS/OAuthMobile.swift

```diff
@@ -343,10 +345,51 @@ open class MobileSharedApplication: SharedApplication {
     }
 
     open func presentAuthChannel(_ authURL: URL, tryIntercept: @escaping ((URL) -> Bool), cancelHandler: @escaping (() -> Void)) {
-        if let controller = self.controller {
-            let safariViewController = MobileSafariViewController(url: authURL, cancelHandler: cancelHandler)
-            controller.present(safariViewController, animated: true, completion: nil)
+        guard self.authChannel == nil else {
+            return
+        }
```

Comment:
we're in a singleton here, so don't start multiple sessions on top of each other